lib/v2/datasource/remote/model/balance_model.dart

```diff
@@ -15,13 +15,11 @@ class BalanceModel {
   String get roundedQuantity => '${quantity.seedsFormatted}';
 
   factory BalanceModel.fromJson(List<dynamic> json) {
-    if (json.isEmpty || json[0].isEmpty) { 
-      // is a user has no balance, the list returned is empty
-      // not really sure an emtpy string can happen
-      return const BalanceModel(0);
+    if (json.isEmpty || json[0].isEmpty) {
```

Comment:
this check can happen at the repo level, and the repo can return ErrorResult if this is the case.

Reply:
This is not an error. It just means the user doesn't have this token.

Reply:
then the repo returns null.